docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,83 @@
+---
+order: 5
+title: Emergency Upgrades
+---
+
+## **Introduction**
+
+This article goes into the general flow of events for an emergency upgrade, pre-upgrade and during the upgrade for all validators.
+
+Emergency upgrades are an unfortunate necessity for all validators when running a PoS node.
+
+When a vunerability is discovered and responsibily disclosed, the vunerability will be reported to one of the channels described in the [Security.md](../../SECURITY.md).
+
+Once the vulnerability is patched and tested, the core team will reach out to all validators to update their nodes with the latest version.
+
+### **Cosmos Hub Upgrade Information**
+
+Depending on the severity of the vulnerability, the core teams and validators can take different actions.
+
+The core teams will try reach out **48 hours** or more before a scheduled upgrade, so that validators have time to inspect the changes and also adjust their internal planning accordingly.
```

Comment:
```suggestion
The core teams will try to reach out **48 hours** or more before a scheduled upgrade, so that validators have time to inspect the changes and also adjust their internal planning accordingly.
```

Reply:
The source code is not realeased at this point for security upgrades. The main point of the heads-up communication 24-48 hours before release is to let validators prepare and clear their schedule. (Maybe they can do preliminary steps, like agreeing in a height for coordinated deployments.)

Reviewing the source code starts _after_ the release for security patches.

Reply:
48 hours was the feedback we got from the validator retro - from one of the bigger validators. I've changed to 24, but for bigger validators we might need to release code a little earlier from them to review at a comfortable pace.